doc/source/whatsnew/v0.25.0.rst

```diff
@@ -617,13 +617,12 @@ Strings
 - Improved error message when passing :class:`Series` of wrong dtype to :meth:`Series.str.cat` (:issue:`22722`)
 -
 
-
 Interval
 ^^^^^^^^
 
 - Construction of :class:`Interval` is restricted to numeric, :class:`Timestamp` and :class:`Timedelta` endpoints (:issue:`23013`)
 - Fixed bug in :class:`Series`/:class:`DataFrame` not displaying ``NaN`` in :class:`IntervalIndex` with missing values (:issue:`25984`)
--
+- :class:Interval now refuses to create intervals like `[0,0)` or `(0,0]` (:issue:`26893`)
```

Comment:
- `` :class:`Interval` ``
-  This description is a bit too colloquial. Could you describe the conditions where the new `ValueError` is raised.